_elements/inputs.md

```diff
@@ -6,7 +6,23 @@ title: Inputs
 
 <div class="preview">
   <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Text_Input_UI_v1.png">
+
+  <label for="">Text Input Label</label>
```

Comment:
@msecret we already include `for` attribute, it's not filled though. Do you mean to fill it like:

`<label for="input-type-text">Text Input Focused</label>`
`<input class="usa-input-focus" type="text" name="input-type-text">`